### PR TITLE
Fix login page logo issue

### PR DESCRIPTION
**Pull request by sandeepasingh116**
The company logo was not showing up on the login page and header when ASM was launched via HMC.

Fixed the path of the image.

Defect link : https://jazz07.rchland.ibm.com:13443/jazz/web/projects/CSSD#action=com.ibm.team.workitem.viewWorkItem&id=395134

Signed-off-by: Sandeepa Singh [sandeepa.singh@ibm.com](mailto:sandeepa.singh@ibm.com)

### CODE REVIEW

File: tests/unit/__snapshots__/AppHeader.spec.js.snap [hunk 1]
```diff
@@ -56,7 +56,7 @@ exports[`AppHeader.vue should render correctly 1`] = `
           <img
             alt="Built on OpenBMC"
             class="header-logo"
-            src="@/assets/images/logo-header.svg"
+            src="http://localhost/img/logo-header.svg"
```
Comment (gtmills) [inline tests/unit/__snapshots__/AppHeader.spec.js.snap:59]: This is correct..?

Comment (gtmills) [reply tests/unit/__snapshots__/AppHeader.spec.js.snap:59]: Do we need to change the test?

File: src/components/AppHeader/AppHeader.vue [hunk 2]
```diff
@@ -239,6 +239,9 @@ export default {
       event.preventDefault();
       this.$root.$emit('skip-navigation');
     },
+    getImageUrl() {
```
Comment (gtmills) [inline src/components/AppHeader/AppHeader.vue:242]: does it make more sense to name these getLogoImage()?

File: src/layouts/LoginLayout.vue [hunk 3]
```diff
@@ -34,6 +30,11 @@ export default {
       customizableGuiName: process.env.VUE_APP_GUI_NAME || '',
     };
   },
+  methods: {
+    getImageUrl() {
```
Comment (gtmills) [inline src/layouts/LoginLayout.vue:34]: getLogoImage?

**Review by gtmills — APPROVED** [review]
I had minor comments, address if you agree, if not, I am good.

File: src/components/AppHeader/AppHeader.vue (hunk 2 above)
Comment (sandeepasingh116) [reply src/components/AppHeader/AppHeader.vue:242]: getImageUrl seems fine for now.

File: tests/unit/__snapshots__/AppHeader.spec.js.snap (hunk 1 above)
Comment (sandeepasingh116) [reply tests/unit/__snapshots__/AppHeader.spec.js.snap:59]: This does look a little off but test is automatically updating to this. Will give it one more day to figure out if I can do something about this.

Comment (Nikhil-Ashoka) [reply tests/unit/__snapshots__/AppHeader.spec.js.snap:59]: I just checked without this change and it worked fine, Would undo of this snap file cause any issue?

**Review by Nikhil-Ashoka — APPROVED** [review]

File: src/components/AppHeader/AppHeader.vue [hunk 4]
```diff
@@ -239,6 +239,9 @@ export default {
...
+      return location.origin + location.pathname + 'img/logo-header.svg';
```
Comment (gtmills) [inline src/components/AppHeader/AppHeader.vue:243]: Why didn't assets/images work?

**Pull request merged**